scss/_patterns_navigation-reduced.scss

```diff
@@ -0,0 +1,120 @@
+@mixin vf-p-navigation-reduced {
+  @media (min-width: $breakpoint-navigation-threshold - 1) {
+    .p-navigation--sliding.is-reduced.has-search-open {
+      .p-navigation__nav .p-navigation__items {
+        display: inline-flex;
+      }
+
+      .p-navigation__search {
+        align-items: center;
+        background-color: #262626;
+        display: flex;
+        justify-content: center;
+        left: 0;
+        padding: 0.625rem 1rem;
+        position: absolute;
+        top: 2rem;
+        width: 100%;
+
+        .p-search-box {
+          margin: 0;
+          max-width: 50rem;
+        }
+      }
+
+      .p-navigation.is-secondary {
+        display: none;
+      }
+    }
+  }
+
+  .p-navigation--sliding.is-reduced {
+    position: relative;
+
+    .p-navigation__items:first-child {
+      margin-left: -1rem;
+    }
+
+    .p-navigation__logo-tag {
+      display: none;
+    }
+
+    .p-navigation__tagged-logo {
+      .p-navigation__link {
+        padding-left: 0;
+      }
+    }
+
+    .p-navigation__link,
+    .p-navigation__logo-title {
+      font-size: 0.875rem;
+      color: $color-mid-light;
+      font-weight: 300;
+      transition: none;
+      line-height: 1rem;
+    }
+
+    @media (max-width: $breakpoint-navigation-threshold - 1) {
+
+      .p-navigation__logo-title,
+      .p-navigation__link {
+        color: $color-x-light;
+        font-size: 1rem;
+        line-height: 1.5rem;
+      }
+    }
+
+    @media (min-width: $breakpoint-navigation-threshold - 1) {
+      .p-navigation__link::after {
+        top: 0.5rem;
+      }
+    }
+
+    .p-navigation__link--search-toggle::after {
+      right: 1rem;
+      top: 0.75rem;
+    }
+
+    @media (min-width: $breakpoint-navigation-threshold) {
+      background-color: $color-dark;
+
+      .p-navigation__link {
+        padding: 0.5rem 2rem 0.5rem 1rem;
+      }
+
+      .p-navigation__link--search-toggle {
+        padding-left: 0.5rem;
+
+        &::after {
+          top: 0.5rem;
+        }
+      }
+
+      .p-navigation__item--dropdown-toggle.is-active {
+        background-color: rgba(255, 255, 255, 0.1);
+      }
+    }
+
+    &~.global-nav-dropdown {
+      top: 2.1rem !important;
+    }
+  }
+
+  .p-navigation.is-secondary {
+    position: sticky;
+    top: 0;
+    z-index: 9;
+
+    .p-navigation__nav.is-open {
```

Comment:
Why does it need anything different than standard Vanilla navigation with mobile view?